for-research/phoenix/get_json_phoenix.py

```diff
@@ -290,10 +377,15 @@ def json_format(files: list[str],
                 event_number = processed_events
                 run_number = 1
             else:
-                event_number = events_data['EventID'][processed_events].mc_event_number if isMC>0 else events_data['EventID'][processed_events].event_number
-                # In MC simulation, the run number indicates the conditions used for simulation
-                # The channel number is the dataset ID, which is more intuitive
-                run_number = events_data['EventID'][processed_events].channel_number if isMC>0 else events_data['EventID'][processed_events].run_number
+                if 'EventID' in events_data.fields:
+                    event_id = events_data['EventID'][processed_events]
+                    event_number = event_id.mc_event_number if isMC>0 else event_id.event_number
+                    # In MC simulation, the run number indicates the conditions used for simulation
+                    # The channel number is the dataset ID, which is more intuitive
+                    run_number = event_id.channel_number if isMC>0 else event_id.run_number
+                else:
+                    print("EventID not found in events_data, skipping event.")
+                    continue  # Cannot proceed without event number and run number
```

Comment:
This feels too aggressive to me? That is, if we can't find an EventID, we can just treat it like we do for an ntuple, rather than skipping the event all together?